app/Repositories/Eloquents/BaseRepository.php

```diff
@@ -0,0 +1,106 @@
+<?php
+namespace App\Repositories\Eloquents;
+ 
+use App\Models\Webhook;
+use App\Repositories\Interfaces\BaseRepositoryInterface;
+use Auth;
+
+abstract class BaseRepository implements BaseRepositoryInterface
+{
+    /**
+     * @var \Illuminate\Database\Eloquent\Model
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.PropertyDeclaration.Underscore</br>**Severity**: warning</br>**File**: app/Repositories/Eloquents/BaseRepository.php</br>**Line**: 13</br>Property name "$_model" should not be prefixed with an underscore to indicate visibility